ledger/error/src/main/scala/com/daml/error/ErrorCode.scala

```diff
@@ -138,16 +136,12 @@ abstract class ErrorCode(val id: String, val category: ErrorCategory)(implicit
   ): StatusRuntimeException = {
     val status = asGrpcStatus(err)
     // Builder methods for metadata are not exposed, so going route via creating an exception
-    val ex = StatusProto.toStatusRuntimeException(status)
-    // Strip stack trace from exception
-    // TODO error codes: Define a generic mechanism (trait or method) to check if errors are logged on creation,
-    //                   instead of checking every implementation.
+    val e = StatusProto.toStatusRuntimeException(status)
```

Comment:
Nit: For the sake of readability, can we avoid one-character variable names, please?